packages/dds/sequence/src/revertibles.ts

```diff
@@ -134,6 +159,153 @@ export function appendLocalPropertyChangedToRevertibles(
 	return revertibles;
 }
 
+function addIfIntervalEndpoint(
+	ref: LocalReferencePosition,
+	segmentIndex: number,
+	startIntervals: { segmentIndex: number; interval: SequenceInterval }[],
+	endIntervals: { segmentIndex: number; interval: SequenceInterval }[],
+) {
+	if (refTypeIncludesFlag(ref.refType, ReferenceType.RangeBegin)) {
```

Comment:
When I create local references for reverting change and delete edits, I assign the type to ReferenceType.RangeBegin | ReferenceType.StayOnRemove. Would that cause a problem here, if you're expecting RangeBegin but don't see it because of my creation code?